packages/block/test/testdata/testdata-from-rpc-difficulty-as-integer.json

```diff
@@ -25,77 +20,73 @@
       "r": "0x5d92c10b5789801d4ce0fc558eedc6e6cccbaf0105a7c1f909feabcedfe56cd9",
       "raw": "0xf86d819d8504e3b2920082520894f4702bb51b8270729db362b0d4f82a56bdd66c6588120a871cc0020000801ba05d92c10b5789801d4ce0fc558eedc6e6cccbaf0105a7c1f909feabcedfe56cd9a072cc370fa5fd3b43c2ba4e9e70fea1b5e950b4261ab4274982d8ae15a3403a33",
       "s": "0x72cc370fa5fd3b43c2ba4e9e70fea1b5e950b4261ab4274982d8ae15a3403a33",
       "to": "0xf4702bb51b8270729db362b0d4f82a56bdd66c65",
       "transactionIndex": "0x1",
-      "v": 0,
+      "type": "0x0",
+      "v": "0x1b",
       "value": "0x120a871cc0020000"
     },
     {
       "blockHash": "0xc596cb892b649b4917da8c6b78611346d55daf7bcf4375da86a2d98810888e84",
       "blockNumber": "0x2ca14a",
-      "creates": null,
       "from": "0x1e9939daaad6924ad004c2560e90804164900341",
       "gas": "0x9858",
       "gasPrice": "0x4a817c800",
       "hash": "0x3078eeb8227d104338666de260aac59c141a08f519856fd8b7253398d9347f51",
       "input": "0x",
       "nonce": "0x22f5d",
-      "publicKey": "0x9fd89da696721cb906af81fe5d8bc3d0a4a7368be52db98d9e3a2d7c5f7aef545e8e578c2a158236d6221512c5df892b23bbe1087966f134799d6e36d3097973",
       "r": "0x7ee15b226f6c767ccace78a4b5b4cbf0be6ec20a899e058d3c95977bacd0cbd5",
       "raw": "0xf86f83022f5d8504a817c80082985894b8201140a49b0d5b65a23b4b2fa8a6efff87c576880de4ea09ac8f1e888025a07ee15b226f6c767ccace78a4b5b4cbf0be6ec20a899e058d3c95977bacd0cbd5a027e75bcd3bfd199e8c3e3f0c90b0d39f01b773b3da64060e06c0d568ae5c7523",
       "s": "0x27e75bcd3bfd199e8c3e3f0c90b0d39f01b773b3da64060e06c0d568ae5c7523",
       "to": "0xb8201140a49b0d5b65a23b4b2fa8a6efff87c576",
       "transactionIndex": "0x2",
-      "v": 0,
+      "type": "0x0",
+      "v": "0x1b",
       "value": "0xde4ea09ac8f1e88"
     },
     {
       "blockHash": "0xc596cb892b649b4917da8c6b78611346d55daf7bcf4375da86a2d98810888e84",
       "blockNumber": "0x2ca14a",
-      "creates": null,
       "from": "0xea674fdde714fd979de3edf0f56aa9716b898ec8",
       "gas": "0x15f90",
       "gasPrice": "0x4a817c800",
       "hash": "0x9de43b061e5286ab1ad7494f50fac1ec9b541998800f9388ae6e7119f312c5cd",
       "input": "0x",
       "nonce": "0xfc02d",
-      "publicKey": "0x5d8af02c52b1fb40e45d6e04ae2351d443d1f7327e9b6dbb5ae1d918414899c625690c4c236889e23d9705f9db88876542b959d947a3497a30d0d922ee30015f",
       "r": "0x059934eeace580cc2bdc292415976692c751f0bcb025930bd40fcc31e91208f3",
       "raw": "0xf870830fc02d8504a817c80083015f9094c4f381af25c41786110242623373cc9c7647f3f1880e139507cd50c0188026a0059934eeace580cc2bdc292415976692c751f0bcb025930bd40fcc31e91208f3a077ff34a10a3de0d906a0363b4bdbc0e9a06cb4378476d96dfd446225d8d9949c",
       "s": "0x77ff34a10a3de0d906a0363b4bdbc0e9a06cb4378476d96dfd446225d8d9949c",
       "to": "0xc4f381af25c41786110242623373cc9c7647f3f1",
       "transactionIndex": "0x3",
-      "v": 1,
+      "type": "0x0",
```

Comment:
Really cool clean-up PR, thanks @gabrocheleau! ❤️ 🙂

Short half-nit: so these are legacy txs if I get this right, correct? Now with the `v` values somewhat corrected to 27/28. I wonder about this `type` field: is this an allowed combination, is this confirmed? Respectively do some (all) clients do this (now?), to send back a type 0 for legacy txs? 🤔

I am just wondering because I also wouldn't want to introduce some data combinations in test data where "future generations" [TM] (lol) will assume this to be correct and wonder about the combination. 😋

In case this can (or even: should) be removed, you can just add the task TODO to the tiny-fixes-issue #2105.

Reply:
I can't speak for all clients but at least Infura returns in this format for legacy transactions.  I just pulled one from the dao fork era and it came back with `type: 0x0, v: 0x1b`

Reply:
Ah, interesting.